<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Covid-19 genome scan</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Covid-19-genomic-variation</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Spatial_structure.html">Spatial genetic structure of SARS-CoV-2</a>
</li>
<li>
  <a href="Covid_genome_Scan.html">Covid-19 genome scan</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Covid-19 genome scan</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-08-30
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>Covid-19-genomic-variation/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.1). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20200412code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20200412)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20200412code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20200412)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong4d0f27d"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 4d0f27d </a>
</p>
</div>
<div id="strongRepositoryversionstrong4d0f27d" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 4d0f27d. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  analysis/Covid_genome_Scan.Rmd
    Untracked:  analysis/Spatial_structure.Rmd
    Untracked:  data/2019-nCoV_total_10_04_2020cnv.gds
    Untracked:  data/2019-nCoV_total_10_04_2020snps.gds
    Untracked:  data/2019-nCoV_total_17_04_2020snps.gds
    Untracked:  data/2019-nCoV_total_2020-04-17_cnr.gds
    Untracked:  data/PCACNR.html
    Untracked:  data/PCA_covid19_geo_popres_coord_update_country_code_13_04.RData
    Untracked:  data/Virus_genome_Scan.RData

Unstaged changes:
    Modified:   analysis/_site.yml
    Modified:   analysis/about.Rmd
    Modified:   analysis/index.Rmd
    Modified:   analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with <code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="detecting-loci-under-selection" class="section level2">
<h2>Detecting loci under selection</h2>
<p>In line with the spatial genetic structure analysis, and to detect the genome regions under selection or involved in adaptation, we carried out the PCA-based genome scan on the genome-wide SNPs using pcadapt (Duforet-Frebourg, et al. 2016; Luu, et al. 2017). We filtered the SNPs with minor allele frequency (MAF) &gt;0.05 and kept 5 principal components for use of calculating the Mahalanobis distance to identify the outliers as the candidate loci that might be under selection. We set the threshold at alpha &lt;4E-11 to identify the outliers. In addition, we also carried out the genome scan for all variant sites (CNRs, the copy number of references) to see which genes were under selection. The analysis was carried out using the pcadapt package (Luu, et al. 2017).</p>
<p><img src="figure/Covid_genome_Scan.Rmd/fig1-1.png" width="816" style="display: block; margin: auto;" /> Fig. 1 Screen plot for the variations Fig. 1 shows that the first 5 PCs account for almost the largest variance of the total variations based on the whole genome wide SNPs. <img src="figure/Covid_genome_Scan.Rmd/fig2-1.png" width="816" style="display: block; margin: auto;" /> Fig. 2 Score plot for the variations</p>
<p>Fig. 2 is also the genetic structure presented in <a href="link=%22Spatial_structure.html%22">“Spatial genetic structure of SARS-CoV-2”</a> . <img src="figure/Covid_genome_Scan.Rmd/fig3-1.png" width="816" style="display: block; margin: auto;" /> Fig. 3 Manhattan plot for the variations with a threshold of 4E-11 using the first 5 PCs</p>
<p>This plot presents the -log(q-value) of SNPs under selection. Detailed list of the SNPs corresponding to the genes and positions is presented in our MS.</p>
<p><img src="figure/Covid_genome_Scan.Rmd/fig4-1.png" width="816" style="display: block; margin: auto;" /> Fig. 4 qq-plot for the variations</p>
<p>The genomic position of SARS-Covid-2 under selection:</p>
<pre><code>  [1]    3    5   12   21   23   30   32   33   35   53   54   67   68   70   78
 [16]   83   85   88   89   94   97   99  108  109  110  111  112  119  127  133
 [31]  134  135  136  137  138  140  143  144  146  149  151  152  154  155  160
 [46]  161  163  165  167  172  178  180  181  197  211  217  218  223  232  233
 [61]  234  242  244  245  256  257  261  264  272  274  275  276  278  280  285
 [76]  288  292  297  302  303  317  318  324  328  329  336  339  344  345  354
 [91]  356  361  362  363  368  369  372  376  380  383  385  389  394  395  396
[106]  397  405  406  407  408  413  450  472  475  490  491  496  497  503  505
[121]  523  527  529  532  534  535  536  549  552  553  554  559  562  580  581
[136]  582  586  590  592  624  638  642  654  657  658  673  675  678  680  687
[151]  688  689  696  704  708  714  718  722  723  724  730  738  748  749  753
[166]  754  756  757  759  760  761  762  764  765  767  769  771  773  776  777
[181]  778  779  782  783  784  785  787  789  790  791  792  798  799  801  802
[196]  805  808  812  818  819  833  851  853  855  856  863  864  867  872  876
[211]  878  879  887  894  896  907  909  914  915  917  920  926  942  945  949
[226]  950  952  953  963  967  971  981  996  998  999 1000 1001 1002 1003 1004
[241] 1005 1008 1010 1012 1013 1014 1015 1016 1019 1020 1024 1026 1027 1028 1029
[256] 1037 1044 1045 1046 1048 1051 1058 1063 1066 1068 1069 1076 1077 1080 1081
[271] 1083 1087 1092 1103 1108 1115 1116 1118 1119 1120 1126 1139 1150 1151 1152
[286] 1153 1154 1155 1156 1157 1158 1159 1160 1164 1165 1166 1176 1178 1181 1184
[301] 1186 1187 1188 1191 1198 1201 1204 1207 1210 1219 1222 1223 1224 1225 1227
[316] 1231 1234 1235 1236 1239 1244 1245 1250 1266 1273 1276 1280 1284 1288 1297
[331] 1299 1306 1310 1311 1322 1327 1330 1332 1336 1344 1347 1348 1349 1355 1356
[346] 1361 1371 1372 1375 1379 1381 1384 1385 1393 1400 1401 1404 1412 1423 1426
[361] 1428 1438 1441 1445 1446 1461 1462 1465 1469 1470 1471 1477 1480 1484 1491
[376] 1507 1510 1514 1516 1518 1519 1520 1521 1522 1525 1527 1528 1533 1534 1535
[391] 1536 1537 1543 1548 1550 1555 1557 1558 1560 1563 1566 1568 1574 1576 1577
[406] 1584 1590 1605 1622 1627 1628 1640 1642 1651 1652 1660 1662 1666 1675 1677
[421] 1682 1694 1702 1703 1705 1708 1720 1732 1744 1760 1764 1771 1772 1773 1780
[436] 1787 1789 1797 1803 1807 1819 1821 1828 1830 1837 1839 1846 1856 1858 1861
[451] 1862 1869 1872 1874 1875 1882 1885 1887 1890 1891 1892 1893 1901 1907 1909
[466] 1912 1921 1934 1935 1936 1937 1945 1947 1949 1951 1957 1958 1962 1964 1965
[481] 1974 1993 1995 1997 2004 2008 2010 2020 2022 2023 2027 2028 2029 2039 2043
[496] 2044 2046 2057 2062 2067 2068 2076 2081 2090 2097 2099 2111 2113 2121 2122
[511] 2129 2130 2131 2145 2146 2151 2161 2167 2172 2177 2181 2192 2193 2200 2201
[526] 2204 2205 2225 2226 2227 2228 2230 2231 2233 2234 2238 2247 2251 2255 2259
[541] 2261 2275 2277 2283 2285 2286 2287 2288 2289 2293 2294 2295 2296 2297 2298
[556] 2302 2306 2308 2311 2312 2313 2317 2320 2335 2339 2347 2350 2361 2366 2372
[571] 2373 2379 2380 2384 2387 2394 2395 2396 2397 2398 2399 2400 2405 2406 2410
[586] 2411 2412 2413 2414 2415 2416 2418 2419 2420 2422 2423 2424 2425 2426 2429
[601] 2431 2432 2433 2434 2435 2437 2438 2439 2440 2441 2442 2443 2445 2446 2447
[616] 2448 2449 2450 2451 2452 2455 2456 2457 2458 2461 2462 2463 2464 2465 2466
[631] 2467 2469 2470 2471 2472 2473 2474 2477 2478 2479 2481 2482 2483 2485 2486
[646] 2488 2489 2490 2491 2492 2493 2494 2495 2496 2497 2498 2499 2501 2502 2503
[661] 2505 2508 2511 2512 2513 2514 2516 2519 2524 2538 2541 2545 2546</code></pre>
<p>Detailed list of the SNPs corresponding to the genes and positions is presented in our MS.</p>
<p>This is a method associated with principal components. Therefore, this approach identifies loci that contribute to the genetic sturtcure of 3,736 complete genomes SARS-Covid-2.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19041)

Matrix products: default

locale:
[1] LC_COLLATE=English_United Kingdom.1252 
[2] LC_CTYPE=English_United Kingdom.1252   
[3] LC_MONETARY=English_United Kingdom.1252
[4] LC_NUMERIC=C                           
[5] LC_TIME=English_United Kingdom.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ggplot2_3.2.1   qvalue_2.18.0   pcadapt_4.3.1   workflowr_1.6.1

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.2        plyr_1.8.4        compiler_3.6.1    pillar_1.4.2     
 [5] later_1.0.0       git2r_0.26.1      tools_3.6.1       digest_0.6.21    
 [9] viridisLite_0.3.0 lifecycle_0.1.0   jsonlite_1.6      evaluate_0.14    
[13] tibble_2.1.3      gtable_0.3.0      pkgconfig_2.0.3   rlang_0.4.6      
[17] yaml_2.2.0        xfun_0.11         httr_1.4.1        withr_2.1.2      
[21] stringr_1.4.0     dplyr_0.8.3       knitr_1.26        maps_3.3.0       
[25] vctrs_0.3.1       htmlwidgets_1.5.1 fs_1.3.1          cowplot_1.0.0    
[29] rprojroot_1.3-2   grid_3.6.1        tidyselect_0.2.5  data.table_1.12.6
[33] glue_1.3.1        R6_2.4.0          plotly_4.9.1      PCAviz_0.3-37    
[37] rmarkdown_2.1     tidyr_1.0.0       purrr_0.3.3       reshape2_1.4.3   
[41] magrittr_1.5      splines_3.6.1     backports_1.1.5   scales_1.0.0     
[45] promises_1.1.0    htmltools_0.4.0   assertthat_0.2.1  countrycode_1.1.0
[49] colorspace_1.4-1  httpuv_1.5.2      labeling_0.3      stringi_1.4.3    
[53] lazyeval_0.2.2    munsell_0.5.0     crayon_1.3.4     </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
