---
title: "Spatial genetic structure of SARS-CoV-2"
output:
  workflowr::wflow_html:
    toc: false
editor_options:
  chunk_output_type: console
#date: "`r Sys.Date()`"
update date: "10-04-2020"
---

## An interactive map of global spatial genetic structure of SARS-CoV-2 

```{r setup, include=FALSE,echo=FALSE,message=FALSE,warning=FALSE}
knitr::opts_chunk$set(echo = FALSE)
```
To investigate the spatial genetic variations across these 62 regions, we conducted principal component analysis on the genotypes of these virus samples. We first did the PCA on the bi-allelic markers (SNPs). We exacted 2, 551 bi-allelic markers (SNPs) and conducted the PCA with minor allele frequency (MAF) >0.05 and LD at r2 =0.8 using the SNPrelate package (Zheng, et al. 2012). We also carried out the PCA on all SNPs in order to capture a thorough understanding of genome-wide single-nucleotide polymorphism variations. As this virus was reported to have a high mutation rate (Phan 2020; Shen, et al. 2020), to construct a more comprehensive genetic map across space, we further carried out the principal component analysis on all the variant sites (CNRs, the copy number of references), including bi-allelic SNPs, multi-allelic SNPs, indels and structural variants. To see the genetic structure and the genetic relation of these viruses from some heavily impacted countries as well as the early reported countries, we also presented the genetic variation map for China, America, Australia, and Canada.
We provide interactive spatial genetic maps for two types of markers (SNPs and CNRs) for users to look into the local and global structure of this virus. 


```{r library,echo=FALSE,message=FALSE,warning=FALSE}

library(PCAviz)
library(plotly)
infile <- file.path("data", "PCA_covid19_geo_popres_coord_update_country_code_13_04.RData")
load(infile)
```

### Global spatial genetic structure of SARS-CoV-2 represented by SNPs

```{r fig1-1, fig.height = 10, fig.width = 8.5, fig.align = "center",echo=FALSE,message=FALSE,warning=FALSE} 

#plot the data projection on the components
library(plotly)
library(PCAviz)


plot(PCAsnp_covid19_geo_popres,draw.points =TRUE,group = NULL,show.legend = TRUE, geom.point.params = list(size = 2,na.rm = TRUE),plot.title="Global genetic structure of SARS-CoV-2 (SNPs)")+ theme(legend.position = "bottom")+theme(legend.title = element_blank())+guides(size = 5)
```
Fig. 1. Spatial genetic structure of SARS-CoV-2 based on whole genome wide SNPs

Below is the interactive plot of this population structure.
```{r fig1-2, fig.height = 7, fig.width = 9, fig.align ="center",echo=FALSE,message=FALSE,warning=FALSE} 
plot(PCAsnp_covid19_geo_popres,plotly=TRUE,plot.title="Global genetic structure of SARS-CoV-2 (SNPs)")

```
Readers can use their mouses to zoom in and zoom out to check the samples. Double click the legend will hidde the corresponding points from the plots.

### Global spatial genetic structure of SARS-CoV-2 represented by copy number of variations

```{r fig2-1, fig.height = 10, fig.width = 8.5, fig.align = "center",echo=FALSE,message=FALSE,warning=FALSE} 

plot(PCAcnv_covid19_geo_popres,draw.points =TRUE,group = NULL,show.legend = TRUE, geom.point.params = list(size = 2,na.rm = TRUE),plot.title="Global genetic structure of SARS-CoV-2 (CNR)")+ theme(legend.position = "bottom")+theme(legend.title = element_blank())+guides(size = 5)
```
Fig. 2. Spatial genetic structure of SARS-CoV-2 based on whole genome wide copy number of variations.

Below is the interactive plot of this population structure.
```{r fig2-2, fig.height = 7, fig.width = 9, fig.align = "center",echo=FALSE,message=FALSE,warning=FALSE} 
plot(PCAcnv_covid19_geo_popres,plotly=TRUE,plot.title="Global genetic structure of SARS-CoV-2 (CNR)")
```

### Data source: CNCB- 2019 Novel Coronavirus Resource (2019nCoVR).
